Composer/packages/client/src/components/Page.tsx

```diff
@@ -117,7 +126,20 @@ const Page: React.FC<IPageProps> = (props) => {
         </div>
         <div css={main} role="main">
           <LeftRightSplit initialLeftGridWidth="20%" minLeftPixels={200} minRightPixels={800}>
-            <NavTree navLinks={navLinks} regionName={navRegionName} />
+            {useNewTree ? (
+              <ProjectTree
+                showDelete={false}
+                showTriggers={false}
+                onSelect={(link) => {
+                  navigateTo(buildURL(pageMode, link));
+                }}
+                onSelectAllLink={() => {
+                  navigateTo(buildURL(pageMode, { projectId: rootProjectId, dialogId: '$all' }));
```

Comment:
`$all` seems a little magical to me as well. Why is the `$` prepended?

Reply:
That was because I needed a name that couldn't be a valid dialog name ($ isn't a valid character in those). This is obsolete now, though.